snap/snapcraft.yaml

```
@@ -106,7 +106,7 @@ parts:
       - libboost1.71-dev
       - libapparmor-dev
     stage-packages:
-      - libmiral4
+      - libmiral5
```

This will break the build for the beta channel:

```suggestion
      - try:
          - libmiral5
      - else:
          - libmiral4
```

---

We really need a `stable` branch of some kind…